docs/notebooks/tutorial.ipynb

```diff
@@ -0,0 +1,890 @@
+    "\n",
+    "\n",
+    "Consider the energy function\n",
+    "\n",
+    "$E(s_1,\\dots,s_n) = \\sum_{<i,j>} J_{i,j}s_i s_j + \\sum_i h_i s_i$\n",
+    "\n",
+    "where here each $s_i, J_{i,j}$, and $h_i$ are either +1 or -1. Here each index i is associated with a bit on a square lattice, and the $<i,j>$ notation means sums over neighboring bits on this lattice. The problem we would like to solve is, given $J_{i,j}$, and $h_i$, find an assignment of $s_i$ values that minimize $E$.\n",
+    "\n",
+    "How does a variational quantum algorithm work for this? One approach is to consider $n$ qubits and associate them with each of the bits in the classical problem. This maps the classical problem onto the quantum problem of minimizing the expectation value of the observable\n",
+    "\n",
+    "$H=\\sum_{<i,j>} J_{i,j} Z_i Z_j + \\sum_i h_iZ_i$\n",
+    "\n",
+    "Then one defines a set of parameterized quantum circuits, i.e., a quantum circuit where the gates (or more general quantum operations) are parameterized by some values. This produces an ansatz state\n",
+    "\n",
+    "$|\\psi(p_1, p_2, \\dots, p_k)\\rangle$\n",
+    "\n",
+    "where $p_i$ are the parameters that produce this state (here we assume a pure state, but mixed states are of course possible).\n",
+    "\n",
+    "The variational algorithm then works by noting that one can obtain the value of the objective function for a given ansatz state by\n",
+    "\n",
+    "1. Prepare the ansatz state.\n",
+    "2. Make a measurement which samples from some terms in H.\n",
+    "3. Goto 1.\n",
+    "\n",
+    "Note that one cannot always measure $H$ directly (without the use of quantum phase estimation). So one often relies on the linearity of expectation values to measure parts of $H$ in step 2. One always needs to repeat the measurements to obtain an estimate of the expectation value. How many measurements needed to achieve a given accuracy is beyond the scope of this tutorial, but Cirq can help investigate this question.\n",
+    "\n",
+    "The above shows that one can use a quantum computer to obtain estimates of the objective function for the ansatz. This can then be used in an outer loop to try to obtain parameters for the lowest value of the objective function. For these values, one can then use that best ansatz to produce samples of solutions to the problem, which obtain a hopefully good approximation for the lowest possible value of the objective function.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a circuit on a Grid\n",
+    "\n",
+    "To build the above variational quantum algorithm using Cirq, one begins by building the appropriate circuit. In Cirq, circuits are represented either by a `Circuit` object or a `Schedule` object. \n",
+    "\n",
+    "`Schedule` objects offer more control over quantum gates and circuits at the timing level, which we do not need, so here we will work with `Circuit` objects instead.\n",
```

Comment:
Delete schedule.

Reply:
Done: I replaced everything by the new (`.md`) texts in https://cirq.readthedocs.io/en/stable/tutorial.html